airbyte-db/src/main/java/io/airbyte/db/jdbc/JdbcUtils.java

```diff
@@ -237,4 +240,35 @@ public static JsonSchemaPrimitive getType(JDBCType jdbcType) {
 
   }
 
+  /**
+   * Given a database connection and identifier, adds db-specific quoting.
+   *
+   * @param connection database connection
+   * @param identifier identifier to quote
+   * @return quoted identifier
+   * @throws SQLException throws if there are any issues fulling the quoting metadata from the db.
+   */
+  public static String enquoteIdentifier(Connection connection, String identifier) throws SQLException {
```

Comment:
why not call it quoteIdentifier?

Reply:
i was matching the names that jdbc uses. no preference though. quoteIdentifier works for me.

Reply:
If you're following an existing convention, let's stick to it.